main/load-tx-rlp.zkasm

```diff
@@ -323,7 +324,7 @@ invalidTxRLP:
         $ => E                          :MLOAD(batchHashDataId)
 
 appendTxs:
-        B - C - 32                      :JMPN(finalAppendTxs)
+        B - C - 32                      :JMPN(finalAppendTxs) ; TODO check, JUMPN only chekcs first 64 bits
```

Comment:
```suggestion
        B - C - 32                      :JMPN(finalAppendTxs) ; TODO check, JUMPN only chekcs first 32 bits
```

Reply:
I would remove the comments because the same reason as `load-tx-rlp-utils.zkasm`